pkg/volume/glusterfs/glusterfs.go

```diff
@@ -490,13 +495,15 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 	if len(pv.Spec.AccessModes) == 0 {
 		pv.Spec.AccessModes = r.plugin.GetAccessModes()
 	}
+	sGid := strconv.FormatInt(reqGid, 10)
+	pv.Annotations = map[string]string{"pv.beta.kubernetes.io/gid": sGid}
```

Comment:
@wongma7 @erinboyd at present we use this annotation to avoid communication of SGID between the pod and PV. but if I interpreted @liggitt's comment properly,  the annotations in general  are actually getting deprecated or the plan is to deprecate it in upcoming k8s versions. so, what abt this particular annotation ? are we planning to use it in this 1.5 release ?